src/camera.jl

```diff
@@ -12,8 +12,8 @@ function cubeside_const_lift(_, id, mousehover)
     Vec3f0(1)
 end
 import Base: +
-Base.clamp{T}(x::T) = clamp(x, T(0),T(1))
-+{T}(a::RGBA{T}, b::Number) = RGBA{T}(clamp(comp1(a)+b), clamp(comp2(a)+b), clamp(comp3(a)+b), clamp(alpha(a)+b))
+Base.clamp(x::T) where {T} = clamp(x, T(0),T(1))
```

Comment:
could this use a package-local name? I don't think this signature would be accepted in a base PR, and it could conceal bugs otherwise